frontend/src/react-query/api/user.tsx

```diff
@@ -88,7 +88,7 @@ export const useListUsersQuery = (
   const listUsersResult = useInfiniteQueryWithAllPages(listUsers, listUsersRequest, {
     pageParamKey: 'pageToken',
     enabled: options?.enabled,
-    getNextPageParam: (lastPage) => lastPage.nextPageToken,
+    getNextPageParam: (lastPage) => lastPage.nextPageToken || undefined,
```

Comment:
What does this change help with?

Reply:
this fixes the isue where the query would keep fetching the endpoint [this](https://redpandadata.slack.com/archives/C081G37T71B/p1747333480572309?thread_ts=1747333453.921179&cid=C081G37T71B)

Reply:
btw this endpoint if you set the size to 0 it would load [all users](https://github.com/redpanda-data/console/blob/master/backend/pkg/api/connect/service/user/v1/service.go#L109) though we could run into max payload size issues that is way I left the infinite query